src/DataWrangling/metadata_field.jl

```diff
@@ -57,6 +57,9 @@ function Field(metadata::Metadatum, arch=CPU();
                halo = (3, 3, 3),
                cache_inpainted_data = true)
 
+    # Ensure data is available
```

Comment:
the comment is a little confusing. It's not about "ensuring" anything -- we just need the data

Reply:
#515